metadata-ingestion/tests/unit/urns/valid_urns.txt

```diff
@@ -0,0 +1,24 @@
+# Unknown entity types become generic urns
```

Comment:
I wonder if it makes sense to put this in a more common location. Technically this file can be read from anywere in the repo. So not a blocker just something to consider.

Reply:
Yeah I’m not opinionated on where these files live - when we add the corresponding Java tests, feel free to move them wherever